### DISCUSSION TITLE
Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvException: Error -24 EMFILE too many open files

### DISCUSSION THREAD

**Discussion by tlejmi** [dotnet/aspnetcore#26365] [General]
I'm serving byte[] to more than 3000 users .... Im reading from files and thoses File are generated at the same time . 
using ASP .NET core 3.1 and Ketstrel 
Im facing this Exeption : 
   Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal.Networking.UvException: Error -24 EMFILE too many open files

How can I solve it ....  

dotnet --version  3.1.402

NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.5 LTS"
VERSION_ID="18.04"

**Comment by davidfowl** [comment]
I'm not sure I've ever seen this but you should try not using libuv if you can.
